packages/jest-core/src/SearchSource.ts

```diff
@@ -6,6 +6,8 @@
  */
 
 import path from 'path';
+import ScriptTransformer from '@jest/transform/build/ScriptTransformer';
+import {SnapshotResolver} from 'jest-snapshot/build/snapshot_resolver';
```

Comment:
don't import from `build`, only top level `index`. If that export is missing, add it

Reply:
Good catch! I thought I'd gotten them all, but missed this one 😬

Reply:
unresolved